src/quickfix.c

```diff
@@ -12,6 +12,7 @@
  */
 
 #include "vim.h"
+#include "limits.h"
```

Comment:
this should not be needed as it is in vim.h

Reply:
Ah, so it is! Removed.